examples/repl/main.rs

```diff
@@ -448,6 +448,10 @@ fn handle_cmd(line: &str, ctx: Arc<RwLock<Context>>) -> Result<ExitResult, Error
     let arg1 = args.next().unwrap_or_default();
 
     match arg0 {
+        "close" => {
+            let context = ctx.write().unwrap();
+            context.sql.close(&*context);
```

Comment:
I don't think we really want calling `context.sql.close()` as an example.  You shouldn't need to do this from the API, instead you should drop the entire context and that will make sure sqlite is closed.

Reply:
The current behavior is: The help says that there is a close command but there actually is none. So, you propose to completely remove this? (we should really remove this command from the help then)

Reply:
BTW I just needed it for #1489, maybe it will be useful for testing again in the future.

Reply:
what does dc_close() acutally do? i think a `close` command in the repl-tool should do the same - otherwise is would use a different name, eg. `forcesqlclose`.

Reply:
`dc_close()` drops the context: https://github.com/deltachat/deltachat-core-rust/blob/master/deltachat-ffi/src/lib.rs#L287-294 (the `.take()` changes the option from `Some(context)` to `None`)

Looking at this file here that's a lot more refactoring... probably why it wasn't done when this behaviour was introduced.  My guess is that the help text was overlooked when the close command was removed.  So the easiest thing to do is remove the help text...

If you really want a working close you need to store it in an Option probably, like the ffi does: https://github.com/deltachat/deltachat-core-rust/blob/master/deltachat-ffi/src/lib.rs#L65